vsts/internal-version-increment.sh

```diff
@@ -75,3 +75,8 @@ fi
 if [ "$BUILD_SOURCEBRANCHNAME" != "master" ]; then
   sed -i '' 's/\(VERSION_STRING[[:space:]]*=[[:space:]]*\).*/\1'$publish_version'/g' $VERSION_FILENAME
 fi
+
+# Update swift package manager config file
+swift_package_file='Package.swift'
```

Comment:
Should it be in env variable? Do we have to use `Package.swift`?

Reply:
No, it's a constant, this name is predefined